src/objects/avatar/BaseAvatar.ts

```diff
@@ -121,6 +129,8 @@ export class BaseAvatar extends PIXI.Container {
     this.zIndex = options.zIndex;
     this._nextLookOptions = options.look;
     this._onLoad = options.onLoad;
+    this._skipBodyParts = options.skipBodyParts || false;
+    this._headOnly = options.headOnly || false;
```

Comment:
Doesn't make much difference, but in the code base we usually use `??` instead of `||`.